src/screens/ProfileScreen.tsx

```diff
@@ -1,186 +1,26 @@
-import { useAuthControllerDelete } from '@baca/api/query/auth/auth'
-import { ControlledField } from '@baca/components'
-import { isWeb } from '@baca/constants'
-import { Button, Text, Spacer, Row, Box, useBottomSheet } from '@baca/design-system'
-import {
-  useCallback,
-  useTranslation,
-  useUpdateProfileForm,
-  useScreenOptions,
-  useWeb,
-} from '@baca/hooks'
-import { signOut } from '@baca/store/auth'
-import { showErrorToast } from '@baca/utils'
-import { useRouter } from 'expo-router'
+import { ProfileDeleteAccountButton } from '@baca/components/screens/profile/ProfileDeleteAccountButton'
+import { ProfileDetailsForm } from '@baca/components/screens/profile/ProfileDetailsForm'
```

Comment:
should be imported from '@baca/components'

Reply:
Everything done and commited, all these comments help me a lot to keep improving, I'm really thankful 😊